File: LeetCode/125.Valid_Palindrome/main.py
#!/usr/bin/python
from solution import Solution

# Unit Test
string1 = "A man, a plan, a canal: Panama"
string2 = "race a car"
s = Solution()
if (s.isPalindrome(string1)==True):
    print("PASS")
else:
    print("TEST 1 FAILED")

if (s.isPalindrome(string2)==False):
    print("PASS")
else:
    print("TEST 2 FAILED")

File: LeetCode/125.Valid_Palindrome/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    string string1 = "A man, a plan, a canal: Panama";
    string string2 = "race a car";
    
    if (s.isPalindrome(string1) == true) { cout << "Test 1 - PASS" << endl;}
    else { cout << "Test 1 - Failed" << endl; }
    if (s.isPalindrome(string2) == false) { cout << "Test 2 - PASS" << endl;}
    else { cout << "Test 2 - Failed" << endl; }
    return 0;
}